src/glue_biscuit/schema_utils.py

```diff
@@ -182,20 +180,27 @@ def wrapper_func(*args, **kwargs):
             # Do we want to assume this is right here,
             # or do we want to do crazy reflection magic
             # to figure out a keyword arg? Both?
-            return func(fitted_frame, *kwargs, **kwargs)
+            return func(fitted_frame, *args, **kwargs)
 
         return wrapper_func
 
     return annotation_func
 
 
-def sink(database: str, table: str, schema_obj=None, schema_func=None):
-    spark = SparkSession.builder.getOrCreate()
-    glue_context = GlueContext(spark.sparkContext)
+def __find_glue_context(*args, **kwargs) -> GlueContext:
+    for arg in args:
+        if isinstance(arg, GlueContext):
+            return arg
+    for k, arg in kwargs:
+        if isinstance(arg, GlueContext):
+            return arg
+
```

Comment:
I didn't want to force anyone to use anything in any particular position, but if this is too much magic, let me know.

Reply:
Seems fine.. Maybe should add a `raise` at the end explaining the need for a `GlueContext`